typescript/libs/view_helpers.py

```diff
@@ -213,10 +229,28 @@ def reload_buffer(view, client_info=None):
             client_info.change_count = info.modify_count
         client_info.pending_changes = False
 
+def reload_buffer_on_worker(view):
```

Comment:
There is a number of places we are calling reload_buffer but aren't calling reload_buffer_on_worker.